test/kata6.joinNames.test.js

```diff
@@ -1,5 +1,27 @@
 const { joinNames } = require("../src");
 
+const nameList = [{
+  name: 'Homer'
+}, {
+
+  name: 'Marge'
+
+}, {
+
+  name: 'Bart'
+
+}, {
+
+  name: 'Lisa'
+
+}, {
+
+  name: 'Maggie'
+
+}];
+
 describe("joinNames", () => {
-  test("returns string of names, seperated by commas and an ampersand", () => {});
+  test("returns string of names, seperated by commas and an ampersand", () => {
+    expect(joinNames(nameList)).toEqual('Homer, Marge, Bart, Lisa & Maggie')  
```

Comment:
It would be useful to add extra tests for what would happen if you pass in an empty array, or an array with only one name?